docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -21,7 +21,7 @@ NOTE: If you are looking to use the JSR-107, aka `javax.cache` API, you should s
 [[configuring-with-java]]
 === Configuring with Java
 
-Java configuration is most easily achieved through the use of builders that offer a fluent API.
+Java configuration is most easily achieved through the use of builders that offer a straightforward API.
```

Comment:
fluent is the technical term for the kind of interface we use (`build.withThis().thanThat()`). So I think we want to keep fluent here